skeleton/aws/README.md

```diff
@@ -0,0 +1,84 @@
+## Deploy your infrastructure
+
+You can deploy your infrastructure with the [terraform CLI](https://learn.hashicorp.com/collections/terraform/cli) (`brew install terraform`), or directly using the [Terraform Cloud](https://cloud.hashicorp.com/products/terraform).
+
+This template remains generic to provide more flexibility.
+A workspace and folder strategy needs to be defined first:
+- What resources need to be global?
+- What resources need to be specific to an environment?
+- What workspaces will you need?
+
+While advanced users won't need it, the next part provides a "starter kit" architecture that suits most of the new projects.
+
+## Starter kit
+
+The starter kit has 3 workspaces:
+- **Shared**, that handles a single ECR (Elastic Container Registry) for the whole infrastructure
+- **Staging** and **Prod**: Both workspaces are similar but use different resource names (using the suffixes `-staging` and `-prod`)
+
+### Step 1, Source Code
```

Comment:
This step can be removed once we implement https://github.com/nimblehq/infrastructure-templates/issues/85, right?

Reply:
Yes, definitely! ;-)